tasks/main.yml

```diff
@@ -236,14 +236,18 @@
           - "mssql_password"
           - "mssql_edition"
 
+    # Setting MSSQL_SA_PASSWORD inline for security because setting it with
+    # environment: reveals the value when running playbooks with high verbosity
     - name: Set up MSSQL
-      command: "{{ __mssql_conf_cli }} -n setup"
+      shell: >-
```

Comment:
Out of curiosity, when we use the shell module, we tend to start it with `set -euo pipefail`. It's not required in this pr?

Reply:
I see. Setting environment variables is not counted as a command... Thanks for the clarification.

Reply:
I wish I could use command, but shell is required here for setting environment variable inline.

Reply:
Well, I understood the needs of `shell` here. I'm rather wondering about cases like this... If I search with "shell", there are multiple of running a single command.
https://github.com/linux-system-roles/mssql/pull/159/files#diff-3f874e113ab85f6a2e208aa3904191ec51641e0b5bcfe4123c4a7cf3d67f4a10R80-R81